loopd/config.go

```diff
@@ -86,3 +94,29 @@ func DefaultConfig() Config {
 		},
 	}
 }
+
+// CleanAndExpandPath expands environment variables and leading ~ in the
+// passed path, cleans the result, and returns it.
+// This function is taken from https://github.com/btcsuite/btcd
+func CleanAndExpandPath(path string) string {
```

Comment:
Since we pull in lnd anyway, can't we just use the function `lncfg.CleanAndExpandPath`?

Reply:
indeed!